homeassistant/components/peco/strings.json

```diff
@@ -3,12 +3,26 @@
     "step": {
       "user": {
         "data": {
-          "county": "County"
+          "county": "County",
+          "phone_number": "Phone Number"
+        },
+        "data_description": {
+          "county": "County used for outage number retrieval",
+          "phone_number": "Phone number associated with the PECO account"
```

Comment:
Please better explain this is optional, and the benefit of adding the phone number